Submissions/@palazard.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Pauline Alazard",
+    "img": "https://avatars.githubusercontent.com/u/69105530?v=4",
+    "email": "alazard.pauline@gmail.com",
+    "links": {
+      "website": "",
+      "linkedin": "https://www.linkedin.com/in/pauline-alazard",
+      "github": "https://github.com/palazard"
+    },
+    "jobTitle": "Full Stack Developer",
+    "location": {
+      "city": "from France",
+      "state": "",
+      "country": "Worldwide!"
```

Comment:
please provide the country you are in

Reply:
I'm not always in the same country.. I'm from France but not there all year long.. that why I didn't put a city.